docs/api/en/materials/MeshLambertMaterial.html

```diff
@@ -14,101 +14,122 @@ <h1>[name]</h1>
 		<p class="desc">
 			A material for non-shiny surfaces, without specular highlights.<br /><br />
 
-			The material uses a non-physically based [link:https://en.wikipedia.org/wiki/Lambertian_reflectance Lambertian]
-			model for calculating reflectance. This can simulate some surfaces (such as untreated wood or stone) well,
-			but cannot simulate shiny surfaces with specular highlights (such as varnished wood). [name] uses per-fragment shading.<br /><br />
-
-			Due to the simplicity of the reflectance and illumination models, performance will be greater
-			when using this material over the [page:MeshPhongMaterial],	[page:MeshStandardMaterial] or [page:MeshPhysicalMaterial],
-			at the cost of some graphical accuracy.
+			The material uses a non-physically based
+			[link:https://en.wikipedia.org/wiki/Lambertian_reflectance Lambertian]
+			model for calculating reflectance. This can simulate some surfaces (such
+			as untreated wood or stone) well, but cannot simulate shiny surfaces with
+			specular highlights (such as varnished wood). [name] uses per-fragment
+			shading.<br /><br />
+
+			Due to the simplicity of the reflectance and illumination models,
+			performance will be greater when using this material over the
+			[page:MeshPhongMaterial], [page:MeshStandardMaterial] or
+			[page:MeshPhysicalMaterial], at the cost of some graphical accuracy.
 		</p>
 
-		<iframe id="scene" src="scenes/material-browser.html#MeshLambertMaterial"></iframe>
+		<iframe
+			id="scene"
+			src="scenes/material-browser.html#MeshLambertMaterial"
+		></iframe>
 
 		<script>
+			// iOS iframe auto-resize workaround
 
-		// iOS iframe auto-resize workaround
-
-		if ( /(iPad|iPhone|iPod)/g.test( navigator.userAgent ) ) {
-
-			const scene = document.getElementById( 'scene' );
-
-			scene.style.width = getComputedStyle( scene ).width;
-			scene.style.height = getComputedStyle( scene ).height;
-			scene.setAttribute( 'scrolling', 'no' );
-
-		}
+			if (/(iPad|iPhone|iPod)/g.test(navigator.userAgent)) {
+				const scene = document.getElementById("scene");
 
+				scene.style.width = getComputedStyle(scene).width;
+				scene.style.height = getComputedStyle(scene).height;
+				scene.setAttribute("scrolling", "no");
+			}
```

Comment:
I'm afraid this code is now formatted in the wrong code style.

I've also realized the code section in the geometry pages is now messed up. Can you please fix them?